src/main.rs

```diff
@@ -0,0 +1,253 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_use(value_t)]
+extern crate clap;
+
+use clap::{App, AppSettings, Arg, SubCommand};
+use manticore::io::StdWrite;
+use manticore::protocol;
+use manticore::protocol::firmware_version;
+use manticore::protocol::wire::FromWire;
+use manticore::protocol::wire::ToWire;
+use manticore::protocol::CommandType;
+use manticore::protocol::Header;
+use serde_json;
+use std::fs::OpenOptions;
+use std::io::prelude::*;
+use std::io::BufReader;
```

Comment:
Could you split up these `use` statements by crate?

Reply:
Done.